File: src/main/java/uk/co/threebugs/mochiwhattotrade3/accountsetupgroups/AccountSetupGroupsFileDto.java
package uk.co.threebugs.mochiwhattotrade3.accountsetupgroups;


import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

@Builder
@Jacksonized
@Value
public class AccountSetupGroupsFileDto {
    String name;
    @JsonProperty("setup-group-name")
    String setupGroupName;
    @JsonProperty("metatrader-account")
    String metatraderAccount;
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/accountsetupgroups/AccountSetupGroupsFileDto.kt
package uk.co.threebugs.mochiwhattotrade3.accountsetupgroups

import com.fasterxml.jackson.annotation.JsonProperty


data class AccountSetupGroupsFileDto(val name: String,
                                     @JsonProperty("setup-group-name") val setupGroupName: String,
                                     @JsonProperty("metatrader-account") val metatraderAccount: String
)
